data-migration/migrate-sql-file-tidb.md

```diff
@@ -0,0 +1,95 @@
+
+* **Method 1**: create the target table schema using TiDB Lightning.
+
+    1. Write SQL files that contain the required DDL statements.
+
+        - The format of file name is `${db_name}-schema-create.sql`, and this file should have `CREATE DATABASE` statements.
+        - The format of file name is `${db_name}.${table_name}-schema.sql`, and this file should have `CREATE TABLE` statements.
+
+    2. During the migration, add the following configuration in `tidb-lightning.toml`:
+
+        ```toml
+        [mydumper]
+        no-schema = false # To create the target table schema using Lightning, set the value to false
+        ```
+
+* **Method 2**: create the target table schema manually.
+
+    During the migration, add the following configuration in `tidb-lightning.toml`:
+
+    ```toml
+    [mydumper]
+    no-schema = true # If you have already created the target table schema, set the value to true, which means skipping the schema creation.
+    ```
+
+## Step 3: Create the configuration file
+
+Create a `tidb-lightning.toml` file with the following content:
+
+{{< copyable "" >}}
+
+```toml
+[lightning]
+# Log
+level = "info"
+file = "tidb-lightning.log"
+
+[tikv-importer]
+# "local"：Default. The local backend is used to import large volumes of data (1 TiB or above). During the import, the target TiDB cluster cannot provide any service.
+# "tidb"：The "tidb" backend can also be used to import small volumes of data (below 1 TiB). During the import, the target TiDB cluster can provide service normally. For the information about backend mode, refer to [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends)
+backend = "local"
+# Set the temporary storage directory for the sorted Key value files. The directory must be empty, and the storage space must be enough to hold the largest single table from the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
+sorted-kv-dir = "${sorted-kv-dir}"
+
+[mydumper]
+# Directory of the data source, supports local path (like `/data/my_datasource/`) or S3 path (like `s3://bucket-name/data-path`).
+data-source-dir = "${my_datasource}"
+
+# Do not create a table schema. If you have manually created the target table schema in #Step 2, set it to true; otherwise, it is false.
+no-schema = true
+
+# The information of target cluster.
+host = "${ip}"
+port = 4000
+user = "${user_name}"
+password = "${password}"
+# The table schema is obtained from the TiDB "status port".
+status-port = ${port}       # For example, 10080
+# Address of the PD cluster
+pd-addr = "${ip}:${port}"   # For example: 172.16.31.3:2379. When backend = "local", you must specify status-port and pd-addr. Otherwise, the import will be abnormal.
```

Comment:
```suggestion
[tidb]
# The information of target cluster
host = ${host}                # For example, 172.16.32.1
port = ${port}                # For example, 4000
user = "${user_name}"         # For example, "root"
password = "${password}"      # For example, "rootroot"
status-port = ${status-port}  # During the import process, lightning needs to obtain table schema information from the "Status Port" of TiDB, such as 10080
pd-addr = "${ip}:${port}"     # The address of the cluster PD. Lightning obtains some information through PD, such as 172.16.31.3:2379. When backend = "local", you must specify status-port and pd-addr. Otherwise, the import will be abnormal.
```